ci/linkcheck.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 
 if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then
-  echo "Doing full link check."
   FLAGS=""
-else
-  echo "Checking files changed in commit range: $TRAVIS_COMMIT_RANGE"
-  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r $TRAVIS_COMMIT_RANGE | tr '\n' ' ')
 
-  echo "Checking only changed files: $CHANGED_FILES"
+  echo "Doing full link check."
+else
+  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r master..HEAD | tr '\n' ' ')
```

Comment:
Sorry, I wasn't clear - this won't work in travis because it will be the master of _your_ fork, not the master of the upstream codebase. So in CI it still needs to use TRAVIS_COMMIT_RANGE. https://gitlab.com/YottaDB/DB/YDB/-/merge_requests/840#note_418423834

Reply:
Even though we rebase first?

https://github.com/rust-lang/rustc-dev-guide/blob/d4581a56ae7b16c258b7e43b21d9caacc66e9ff4/.travis.yml#L18

Reply:
Hmm, are you sure `origin` here is `rust-lang/rustc-dev-guide` and not the fork? If so this seems fine.

Reply:
That said, I'm not super confident...

Reply:
I just changed it to origin/master, so I think this should work properly, but let's just watch the first few PRs and make sure that the right set of files is checked.

Reply:
Hmm... well, it seems like it didn't work...